src/utils/Utils.tsx

```diff
@@ -1 +1,19 @@
-export const BaseUrl: string = "https://upskilling-egypt.com:3003/api/v1";
+import axios from "axios";
+import { toast } from "react-toastify";
+
+export const baseUrl: string = "https://upskilling-egypt.com:3003/api/v1";
+
+export const getRequestHeaders = () => {
+  return { Authorization: `Bearer ${localStorage.getItem("token")}` };
+};
+export const handleApiError = (error: unknown) => {
+  const errMsg =
+    axios.isAxiosError(error) && error.response?.data?.message
```

Comment:
The error will not always be in the same format. It should be if the backend is consistent with formatting error messages, but this might not be the case.